robot_smach_states/test/manipulation/test_manipulation.py

```diff
@@ -22,7 +22,7 @@ def setUp(self):
         self.arm_ds = ds.OccupiedArmDesignator(self.robot, {"required_goals": ['handover_to_human', 'reset']})
 
     def test_handover_to_human(self):
-        state = states.HandoverToHuman(self.robot, self.arm_ds)
+        state = states.manipulation.HandoverToHuman(self.robot, self.arm_ds)
```

Comment:
For uniformity I would prefer to see these states defined in the imports as well